--- internal/pipe/gentoo/gentoo.go
+++ internal/pipe/gentoo/gentoo.go
@@ -73,6 +73,15 @@
	if g.Category == "" {
		g.Category = "app-misc"
	}
+	for i, maintainer := range g.Maintainers {
+		if maintainer.Type == "" {
+			g.Maintainers[i].Type = "person"
+		}
+		t := g.Maintainers[i].Type
+		if t != "person" && t != "project" {
+			return fmt.Errorf("invalid gentoo maintainer type %q: must be person or project", t)
+		}
+	}
	return nil
 }
