--- internal/pipe/gentoo/metadata.go
+++ internal/pipe/gentoo/metadata.go
@@ -197,11 +197,18 @@
		if rid.Type == "" || rid.ID == "" {
			continue
		}
-		node := upstream.elementByAttr("remote-id", "type", rid.Type)
-		if node != nil && node.textContent() == rid.ID {
-			continue
+
+		found := false
+		for _, child := range upstream.elements("remote-id") {
+			for _, attr := range child.attrs {
+				if attr.Name.Local == "type" && attr.Value == rid.Type && child.textContent() == rid.ID {
+					found = true
+					break
+				}
+			}
		}
-		child := metadataTextElement("remote-id", rid.ID)
-		child.setAttr("type", rid.Type)
-		upstream.children = append(upstream.children, child)
+		if !found {
+			child := metadataTextElement("remote-id", rid.ID)
+			child.setAttr("type", rid.Type)
+			upstream.children = append(upstream.children, child)
+		}
	}
 }
